data_steward/cdr_cleaner/cleaning_rules/truncate_fitbit_data.py

```diff
@@ -4,51 +4,44 @@
 Original Issue: DC-1046
 
 The intent is to ensure there is no data after the cutoff date for participants in
-Activity Summary, Heart Rate Minute Level, Heart Rate Summary, and Steps Intraday tables
-by sandboxing the applicable records and then dropping them.
+the fitbit tables by sandboxing the applicable records and then dropping them.
 """
 
 # Python Imports
 import logging
 from datetime import datetime
 
 # Project Imports
-import common
-import constants.cdr_cleaner.clean_cdr as cdr_consts
+from common import JINJA_ENV, FITBIT_TABLES
+from resources import validate_date_string, fields_for
+from constants.cdr_cleaner.clean_cdr import FITBIT, DESTINATION_TABLE, DESTINATION_DATASET, QUERY, DISPOSITION
 from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
 from constants.bq_utils import WRITE_TRUNCATE
 
 LOGGER = logging.getLogger(__name__)
 
-FITBIT_DATE_TABLES = [
-    common.ACTIVITY_SUMMARY, common.HEART_RATE_SUMMARY, common.SLEEP_LEVEL,
-    common.SLEEP_DAILY_SUMMARY
-]
-FITBIT_DATETIME_TABLES = [common.HEART_RATE_MINUTE_LEVEL, common.STEPS_INTRADAY]
-
-FITBIT_TABLES_DATE_FIELDS = {
```

Comment:
I like this idea - removing hard-coding columns and tables for the CR.
Great refactoring!